fab/fab/fabfile.py

```diff
@@ -254,7 +253,7 @@ def env_common():
     # if no server specified, just don't run pillowtop
     pillowtop = servers.get('pillowtop', [])
 
-    deploy = servers.get('deploy', servers['postgresql'])[:1]
+    deploy = servers.get('deploy', servers['webworkers'])[:1]
```

Comment:
May be nicer to have an explicit deploy machine specified in the inventory, but this seems good enough for now